content/community/twitter.md

```diff
@@ -1,10 +1,10 @@
 +++
-title = "Twitter"
+title = "X"
 # https://en.wikipedia.org/wiki/Twitter
-description = "Twitter is a microblogging and social networking service on which users post and interact with messages known as \"tweets\""
+description = "X, is a microblogging and social networking service on which users post and interact with messages known as \"tweets\""
 date = "2022-07-23T03:51:55-04:00"
 draft = false
-logo = "https://upload.wikimedia.org/wikipedia/commons/4/4f/Twitter-logo.svg"
+logo = "https://upload.wikimedia.org/wikipedia/commons/c/ce/X_logo_2023.svg"
```

Comment:
This is using Wikipedia as a CDN, which might not be very reliable. The file name and path might get changed any time. For example, the Twitter logo has already been removed. Also, it might load slower than a local file.

Reply:
I agree with the idea of the local files wherever we can, the official twitter logo can be be downloaded [here](https://about.twitter.com/en/who-we-are/brand-toolkit)